dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ResourcesServiceImpl.java

```diff
@@ -491,87 +470,6 @@ public Result<Object> updateResource(User loginUser,
-                    putMsg(result, Status.RESOURCE_IS_USED);
-                    return result;
-                }
-            }
-
-            for (ResourcesTask existResource : existResourcesList) {
-                int taskId = existResource.getTaskId();
-                long taskCode = taskDefinitionMapper.selectById(taskId).getCode();
-
-                List<ProcessTaskRelation> processTaskRelation = processTaskRelationMapper.selectByMap(
-                        Collections.singletonMap("post_task_code", taskCode));
-                if (processTaskRelation.size() > 0) {
-                    long processDefinitionCode = processTaskRelation.get(0).getProcessDefinitionCode();
-                    int processDefinitionVersion = processTaskRelation.get(0).getProcessDefinitionVersion();
-                    List<ProcessTaskRelation> taskRelationList = processTaskRelationMapper.queryByProcessCode(
-                            processTaskRelation.get(0).getProjectCode(),
-                            processDefinitionCode);
-
-                    List<TaskDefinition> taskDefinitionLogList = new ArrayList<>();
-
-                    if (taskRelationList.size() > 0) {
-                        ProcessDefinitionLog processDefinition =
-                                processDefinitionLogMapper.queryByDefinitionCodeAndVersion(
-                                        processDefinitionCode, processDefinitionVersion);
-                        for (ProcessTaskRelation taskRelation : taskRelationList) {
-                            long taskCodeInProcess = taskRelation.getPostTaskCode();
-                            TaskDefinition taskDefinition = taskDefinitionMapper.queryByCode(taskCodeInProcess);
-                            if (taskCodeInProcess == taskCode) {
-                                // originFullName is a prefix if isDirectory is true
-                                taskDefinition.setTaskParams(RemoveResourceFromResourceList(originFullName,
-                                        taskDefinition.getTaskParams(),
-                                        resource.isDirectory()));
-                                // if isDirectory is true, fullName is the new prefix. we replace old prefix
-                                // of resource fullname with the new prefix.
-                                // if isDirectory is false, fullName is the new path.
-                                taskDefinition.setTaskParams(AddResourceToResourceList(originFullName,
-                                        fullName,
-                                        existResource.getFullName(),
-                                        taskDefinition.getTaskParams(),
-                                        resource.isDirectory()));
-                            }
-                            taskDefinitionLogList.add(taskDefinition);
-                        }
-
-                        // update workflow & task definition associated to the resource
```

Comment:
**MORE EXPLANATION:** We should better not update relative task params info in resource management (when deleting resource), so these part codes are useless in my view.